### ISSUE TITLE
Quiet configuration

### ISSUE THREAD

**Issue by delanym** [sormuras/junit-platform-maven-plugin#112]
Is there a parameter to control this output?

![image](https://github.com/sormuras/junit-platform-maven-plugin/assets/5310238/7c0f204a-5df3-4669-a2ab-a606a78cef94)

**Comment by sormuras** [comment]
Do you mean to suppress the entire output - or just the `WARNING: ...`?

**Comment by delanym** [comment]
So we've got a banner, a warning, a summary, and test results.

Ideally I want control over each of these individually AND collectively.

**Comment by sormuras** [comment]
Except for the warning, you have control over the output using the `details` tweak. If that's not enough, try setting `additionalLauncherOptions` (for the Java Executor) or have full control over the launcher using the `overrideLauncherOptions`.

The warning is due to a missing `execute` in the command line for the launcher and needs a fix in the plugin.

**Comment by delanym** [comment]
What `details` tweak? https://github.com/sormuras/junit-platform-maven-plugin/#plugin-configuration-tweaks
Perhaps you could include a Maven site to produced standardized documentation like https://maven.apache.org/surefire/maven-surefire-plugin/

**Comment by sormuras** [comment]
> What details tweak?

https://github.com/sormuras/junit-platform-maven-plugin/blame/e5dc3527002fc5c14b87a890d716ced08b92d09a/src/main/java/de/sormuras/junit/platform/maven/plugin/Tweaks.java#L32-L33

**Comment by delanym** [comment]
Ok thanks. Banner, results, summary gone. I tried hiding the warning with
https://www.slf4j.org/legacy.html#jul-to-slf4jBridge
```xml
          <dependencies>
            <dependency>
              <groupId>org.slf4j</groupId>
              <artifactId>jul-to-slf4j</artifactId>
              <version>2.0.9</version>
            </dependency>
          </dependencies>
```
but admittedly I'm just guessing

**Comment by sormuras** [comment]
Yeah. Tuning the logger used by JUnit to only emit errors is way to mute the warning.

An actual fix just landed in this plugin's code via https://github.com/sormuras/junit-platform-maven-plugin/commit/b9bbccb72ac0e1b072884d5ec7ffcbb27305262d here:

https://github.com/sormuras/junit-platform-maven-plugin/blob/b9bbccb72ac0e1b072884d5ec7ffcbb27305262d/src/main/java/de/sormuras/junit/platform/maven/plugin/JavaExecutor.java#L210

**Comment by delanym** [comment]
What do you wait for to release?
